ci/cirrus_general_ci.yml

```diff
@@ -12,8 +27,12 @@ linux_aarch64_test_task:
     cpu: 4
     memory: 16G
 
+  <<: *MODIFIED_CLONE
```

Comment:
Interesting syntax, I haven't seen this before. Is this general yaml, or a Cirrus CI thing?

Reply:
I think this is a general yaml thing, but I only encountered this for the first time with cirrus config files.

Reply:
https://stackoverflow.com/questions/6651275/what-do-the-mean-in-this-database-yml-file